crates/lib/src/qpu/api.rs

```diff
@@ -125,6 +126,79 @@ pub async fn submit(
         .ok_or_else(|| GrpcClientError::ResponseEmpty("Job Execution ID".into()))?)
 }
 
+/// Cancel all given jobs that have yet to begin executing.
+///
+/// This action is *not* atomic, and will attempt to cancel every job even when some jobs cannot be
+/// cancelled. A job can be cancelled only if it has not yet started executing.
+///
+/// Success response indicates only that the request was received. Cancellation is not guaranteed,
+/// as it is based on job state at the time of cancellation, and is completed on a best effort
+/// basis.
+///
+/// # Arguments
+/// * `quantum_processor_id` - The quantum processor to execute the job on. This parameter
+///      is required unless using [`ConnectionStrategy::EndpointId`] in `execution_options`
+///      to target a specific endpoint ID.
+/// * `job_ids` - The [`JobId`]s to cancel.
+/// * `client` - The [`Qcs`] client to use.
+/// * `execution_options` - The [`ExecutionOptions`] to use. If the connection strategy used
+///       is [`ConnectionStrategy::EndpointId`] then direct access to that endpoint
+///       overrides the `quantum_processor_id` parameter.
+pub async fn cancel_jobs(
```

Comment:
Similar to previous suggestion:

```suggestion
pub async fn request_job_cancellations(
```